build/gulpfile.vscode.js

```diff
@@ -477,6 +477,47 @@ gulp.task(task.define(
 		exportXLFFolderTask
 	)
 ));
+
+// List of ADS extension XLF files that we want to put into the English resource folder.
+const extensionsFilter = filter([
+	"**/admin-tool-ext-win.xlf",
+	"**/agent.xlf",
+	"**/arc.xlf",
+	"**/asde-deployment.xlf",
+	"**/azdata.xlf",
+	"**/azurecore.xlf",
+	"**/azurehybridtoolkit.xlf",
+	"**/big-data-cluster.xlf",
+	"**/cms.xlf",
+	"**/dacpac.xlf",
+	"**/data-workspace.xlf",
+	"**/import.xlf",
+	"**/kusto.xlf",
+	"**/machine-learning.xlf",
+	"**/Microsoft.sqlservernotebook.xlf",
+	"**/mssql.xlf",
+	"**/notebook.xlf",
+	"**/profiler.xlf",
+	"**/query-history.xlf",
+	"**/resource-deployment.xlf",
+	"**/schema-compare.xlf",
+	"**/server-report.xlf",
+	"**/sql-assessment.xlf",
+	"**/sql-database-projects.xlf",
+	"**/sql-migration.xlf",
+	"**/xml-language-features.xlf"
+])
+
+// Copy ADS extension XLFs into English resource folder.
+gulp.task('import-extensions-xlfs', function () {
```

Comment:
This seems to be a part of the "updating localization files" process you documented? It would be great if we could have a single task that runs all the necessary steps (up to the point of submitting a PR) together as well - is that something we're able to do?